Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -422,6 +422,11 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
         return obj;
     }
 
+    public bool WorkspotIsInternal()
```

Comment:
BuildableJobs already keeps a reference to the instance of the class (Furniture/Utility) that creates it, and that class must implement IBuildable, which means they they must have a Weight and a Height property, which means that the if this method was in BuildableJobs, it can be written like:

```
public bool WorkspotIsInternal()
{
    return WorkSpotOffset.x >= 0 && WorkSpotOffset.x < buildable.Width && WorkSpotOffset.y >= 0 && WorkSpotOffset.y < buildable.Height;
}
```

It makes more sense to place it there than in Furniture.